src/atoms/Layout/layout.module.scss

```diff
@@ -45,11 +55,16 @@ $border: 1px solid color('neutral-6');
     [class*='col-'] {
       border-left: $border;
       border-left: $border;
-      padding: ru(.75) ru(.5);
+      padding: 0 ru(.5) ru(1.25);
```

Comment:
also this is what I'm seeing on my renters branch - were you able to replicate in the RCL?
<img width="791" alt="screen shot 2017-05-22 at 4 54 10 pm" src="https://cloud.githubusercontent.com/assets/13224128/26328001/5b2f8004-3f0f-11e7-86f6-a66dbf2f7a23.png">